docs/resources/azure_resource_health_events.md

```diff
@@ -0,0 +1,95 @@
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_resource_health_events` resource block returns all Azure Resource Health Events within a Subscription or for a particular resource.
+```ruby
+describe azure_resource_health_events do
+  #...
+end
+```
+
+```ruby
+describe azure_resource_health_events(resource_group: 'rhctestenv', resource_type: 'Microsoft.Compute/virtualMachines', resource_id: 'rhctestenvV1PI') do
+  #...
+end
+```
+
+## Parameters
+**Note: To list all service health events in a subscription do not provide any parameter**
+
+- To list events for a particular resource than provide the below mandatory parameters. 
+  Even if one of the below parameters is missing than it lists all events for the subscription.  
+
+| Name                           | Description                                                                          |
+|--------------------------------|--------------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in.                          |
+| resource_type                  | The name of the resource type.                                                       |
+| resource_id                    | The unique identifier of the resource.                                               |
+
+- 
+
+## Properties
+
+|Property            | Description                                        | Filter Criteria<superscript>*</superscript> |
+|--------------------|----------------------------------------------------|-----------------|
+| ids                | A list of the unique resource ids.                 | `id`            |
+| names              | A list of names for all the Resources.             | `name`          |
+| types              | A list of types for all the resources.             | `type`          |
+| properties         | A list of Properties all the resources.            | `properties`    |
+
+
+<superscript>*</superscript> For information on how to use filter criteria on plural resources refer to [FilterTable usage](https://github.com/inspec/inspec/blob/master/dev-docs/filtertable-usage.md).
+
+## Examples
+ 
+### Test that There are health events that have service issue
+```ruby
+describe azure_resource_health_events.where{ properties.select{|prop| prop.eventType == 'ServiceIssue' } } do
```

Comment:
Can we add example like this?

describe azure_resource_health_events(resource_group: 'rhctestenv', resource_type: 'Microsoft.Compute/virtualMachines', resource_id: 'rhctestenvV1PI') do
      its('names') { should include('role') }
    end

Reply:
done